proxy.go

```diff
@@ -183,13 +216,19 @@ func (p *Proxy) Start() {
 		}
 	}
 
-	if p.usingConsul {
+	if p.usingConsul || p.usingKubernetes {
+		log.Info("Creating update goroutine")
```

Comment:
s/update/discovery/? (That feels more descriptive to me if I was debugging and less familiar with Veneur)